README.md

```diff
@@ -1,3 +1,5 @@
 # semeio #
 
 Jobs and workflow jobs used in [ERT](https://github.com/equinor/ert).
+
+....
```

Comment:
This was the _meh_.

Reply:
How come? It is the perfect dummy commit 🥳

Reply:
Please use ellipsis …